From 06b9be091b5729840affc2ab31405a7f7787190f Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Wed, 17 Jul 2024 16:36:01 -0700
Subject: [PATCH] mm/damon: move kunit tests to tests/ subdirectory

There was a discussion about better places for kunit test code[1].
Folowwing the conclusion, move kunit tests for DAMON to tests/
subdirectory.

[1] https://lore.kernel.org/CABVgOS=pUdWb6NDHszuwb1HYws4a1-b1UmN=i8U_ED7HbDT0mg@mail.gmail.com

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 mm/damon/core.c                   | 2 +-
 mm/damon/dbgfs.c                  | 2 +-
 mm/damon/sysfs.c                  | 2 +-
 mm/damon/{ => tests}/core-test.h  | 0
 mm/damon/{ => tests}/dbgfs-test.h | 0
 mm/damon/{ => tests}/sysfs-test.h | 0
 mm/damon/{ => tests}/vaddr-test.h | 0
 mm/damon/vaddr.c                  | 2 +-
 8 files changed, 4 insertions(+), 4 deletions(-)
 rename mm/damon/{ => tests}/core-test.h (100%)
 rename mm/damon/{ => tests}/dbgfs-test.h (100%)
 rename mm/damon/{ => tests}/sysfs-test.h (100%)
 rename mm/damon/{ => tests}/vaddr-test.h (100%)

diff --git a/mm/damon/core.c b/mm/damon/core.c
index a1c32becfc73..348ec1362c47 100644
--- a/mm/damon/core.c
+++ b/mm/damon/core.c
@@ -2275,4 +2275,4 @@ static int __init damon_init(void)
 
 subsys_initcall(damon_init);
 
-#include "core-test.h"
+#include "tests/core-test.h"
diff --git a/mm/damon/dbgfs.c b/mm/damon/dbgfs.c
index 51a6f1cac385..5d0599074a18 100644
--- a/mm/damon/dbgfs.c
+++ b/mm/damon/dbgfs.c
@@ -1145,4 +1145,4 @@ static int __init damon_dbgfs_init(void)
 
 module_init(damon_dbgfs_init);
 
-#include "dbgfs-test.h"
+#include "tests/dbgfs-test.h"
diff --git a/mm/damon/sysfs.c b/mm/damon/sysfs.c
index ecf0835e34b8..a49ba683a6b2 100644
--- a/mm/damon/sysfs.c
+++ b/mm/damon/sysfs.c
@@ -1908,4 +1908,4 @@ static int __init damon_sysfs_init(void)
 }
 subsys_initcall(damon_sysfs_init);
 
-#include "sysfs-test.h"
+#include "tests/sysfs-test.h"
diff --git a/mm/damon/core-test.h b/mm/damon/tests/core-test.h
similarity index 100%
rename from mm/damon/core-test.h
rename to mm/damon/tests/core-test.h
diff --git a/mm/damon/dbgfs-test.h b/mm/damon/tests/dbgfs-test.h
similarity index 100%
rename from mm/damon/dbgfs-test.h
rename to mm/damon/tests/dbgfs-test.h
diff --git a/mm/damon/sysfs-test.h b/mm/damon/tests/sysfs-test.h
similarity index 100%
rename from mm/damon/sysfs-test.h
rename to mm/damon/tests/sysfs-test.h
diff --git a/mm/damon/vaddr-test.h b/mm/damon/tests/vaddr-test.h
similarity index 100%
rename from mm/damon/vaddr-test.h
rename to mm/damon/tests/vaddr-test.h
diff --git a/mm/damon/vaddr.c b/mm/damon/vaddr.c
index 58829baf8b5d..50cd06d0d1ca 100644
--- a/mm/damon/vaddr.c
+++ b/mm/damon/vaddr.c
@@ -730,4 +730,4 @@ static int __init damon_va_initcall(void)
 
 subsys_initcall(damon_va_initcall);
 
-#include "vaddr-test.h"
+#include "tests/vaddr-test.h"
-- 
2.39.2

